test/clj/chronograph/handlers/organization_test.clj

```diff
@@ -4,10 +4,13 @@
             [chronograph.fixtures :as fixtures]
             [chronograph.handlers.organization :as organization]
             [clojure.spec.alpha :as s]
-            [clojure.test :refer :all])
+            [clojure.test :refer :all]
+            [chronograph.db.core :as db]
+            [chronograph.domain.invite :as invite]
+            [chronograph.test-utils :as tu])
   (:import (org.postgresql.util PSQLException)))
 
-(use-fixtures :once fixtures/config fixtures/datasource)
+(use-fixtures :once  fixtures/config fixtures/datasource)
```

Comment:
extra space ... did cljfmt add this?

Reply:
No it didn't, and I think cljfmt overlooks this. It doesn't catch everything, sadly.